app/screens/send-bitcoin-screen/send-bitcoin-completed-screen.tsx

```diff
@@ -114,6 +116,52 @@ const SendBitcoinCompletedScreen: React.FC<Props> = ({ route }) => {
     }
   }, [client, feedbackModalShown, LL, showSuggestionModal, navigation, requestFeedback])
 
+  const copyToClipboard = (text: string, message: string) => {
+    Clipboard.setString(text)
+    toastShow({
+      type: "success",
+      message,
+      LL,
+    })
+  }
+
+  const SuccessActionComponent = () => {
+    if (!successAction) return null
+
+    const originalText = successAction.message ?? ""
+    const truncatedText =
+      originalText.slice(0, 300) + (originalText.length > 300 ? "..." : "")
```

Comment:
according to [lud09](https://github.com/lnurl/luds/blob/luds/09.md) the limit is 144 chars